components/DropdownLight/DropdownLight.jsx

```diff
@@ -0,0 +1,71 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { colors, spacing, baseFontSize } from '../shared/theme';
+
+const itemPropType = PropTypes.oneOfType([
+  PropTypes.string,
+  PropTypes.shape({
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    label: PropTypes.string,
+  }),
+]);
+
+const DropList = styled.select`
+  width: 100%;
+  height: 44px;
+  font-size: ${baseFontSize};
```

Comment:
Are you sure that we need this prop?

Reply:
because if the select is not styled it becomes a small menu

Reply:
The initial font is with 13.33px.